pkgs/desktops/gnome/extensions/extensionOverrides.nix

```diff
@@ -47,9 +48,11 @@ super: lib.trivial.pipe super [
 
   (patchExtension "ddterm@amezin.github.com" (old: {
     nativeBuildInputs = [ gobject-introspection wrapGAppsHook ];
-    buildInputs = [ vte ];
+    buildInputs = [ vte libhandy ];
+    postPatch = ''
+      sed -i '1s|^.*$|#!${gjs}/bin/gjs|' bin/com.github.amezin.ddterm
```

Comment:
Could you use `substituteInPlace` with `--replace-fail` so that changes can be easily detected?

Reply:
I tried to modify the original `substituteInPlace` but couldn't figure out how to replace the whole line and not just the occurrence of the string. I think we need to replace the whole line as the shebang is specific to current system and might not be for example `#!/usr/bin/gjs`.

Is there a way to replace the whole line on match?

Reply:
It should always be `#!/usr/bin/env gjs` on eg.o according to the [commit message](https://github.com/ddterm/gnome-shell-extension-ddterm/commit/3191166b71a98deb061b08861284eda2a3a66941).

Reply:
Ahh true. Changed it to use `substituteInPlace`.

Reply:
Please also use `--replace-fail`, `--replace` is deprecated.

Reply:
Done :+1: